model-optimizer/mo/main.py

```diff
@@ -162,9 +170,11 @@ def prepare_ir(argv: argparse.Namespace):
     if argv.legacy_ir_generation and len(argv.transform) != 0:
         raise Error("--legacy_ir_generation and --transform keys can not be used at the same time.")
 
-    ret_code = check_requirements(framework=argv.framework)
-    if ret_code:
-        raise Error('check_requirements exit with return code {}'.format(ret_code))
+    if not new_front_ends or argv.framework not in new_front_ends:
+        ret_code = check_requirements(framework=argv.framework)
```

Comment:
I don't see a requirements file for the pdpd. I assume it should be "requirements_pdpd.txt".

Reply:
I've modified code to check only 'requirements.txt' for cases when C++ frontend is selected. C++ frontends don't require any specific Python packages to be installed